src/Fitter/src/LikelihoodInterface.cpp

```diff
@@ -325,6 +325,17 @@ double LikelihoodInterface::evalFit(const double* parArray_){
     _chi2HistoryTree_->Fill();
   }
 
+  if( std::isnan(_owner_->getPropagator().getLlhBuffer()) or std::isinf(_owner_->getPropagator().getLlhBuffer()) ){
```

Comment:
This code is numerically wrong since infinity is a valid likelihood.  Let's try to get the numerics right and correctly handle the infinities. It's definitely not an error, so throw is wrong.

This is the correct error check 
```
if( std::isnan(_owner_->getPropagator().getLlhBuffer()) {
````

Reply:
Oh I see, I was thinking it would be usefull for the user to get thrown before the fit start but this is true that the Fitter might explore some of these regions in the phase space...

I'll patch it